### PR TITLE
Fix uninitialized local causing rtc failures

### PR DESCRIPTION
**Pull request by riverar**
```
"The variable 'l' is being used without being initialized."

#  Call Site
00 r_core!failwithmessage+0x234
01 r_core!_RTC_UninitUse+0xa4
02 r_core!cmd_print_blocks+0x4f2 [c:\sources\cutter\radare2\libr\core\cmd_print.c @ 2366] 
03 r_core!cmd_print+0x976 [c:\sources\cutter\radare2\libr\core\cmd_print.c @ 3847] 
04 r_core!r_cmd_call+0x249 [c:\sources\cutter\radare2\libr\core\cmd_api.c @ 237] 
05 r_core!r_core_cmd_subst_i+0x3037 [c:\sources\cutter\radare2\libr\core\cmd.c @ 2904] 
06 r_core!r_core_cmd_subst+0x4df [c:\sources\cutter\radare2\libr\core\cmd.c @ 1933] 
07 r_core!r_core_cmd+0x42e [c:\sources\cutter\radare2\libr\core\cmd.c @ 3608] 
08 r_core!r_core_cmd_str+0x5e [c:\sources\cutter\radare2\libr\core\cmd.c @ 3857] 
09 cutter!CutterCore::cmdj+0x73 [c:\sources\cutter\src\cutter.cpp @ 188] 
```

### CODE REVIEW

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -2348,7 +2348,7 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 	bool use_color = r_config_get_i (core->config, "scr.color");
 	int len = 0;
 	int i;
-	int l;
+	int l = 0;
```
Comment (ret2libc) [inline libr/core/cmd_print.c]: I think the initialization should be done at the beginning of the body of the for loop.

Comment (MaskRay) [reply libr/core/cmd_print.c]: `int l = 0;` at the beginning of the body of the for loop

**Comment by radare** [comment]
unnecessary initialization at https://github.com/radare/radare2/pull/11136/files#diff-55abf099b2aa9100745103d790a1ad9cR2369

**Comment by riverar** [comment]
Thanks for your patience!

**Pull request merged**